docs/releases/6.1.txt

```diff
@@ -237,6 +237,10 @@ Management Commands
   ``suggest_on_error`` argument to ``True`` by default on Python 3.14, enabling
   suggestions for mistyped subcommand names and argument choices.
 
+* Management commands now set :class:`~argparse.ArgumentParser`'s ``color``
+  argument to ``True`` by default on Python 3.14+, enabling colorized help text
+  on supported terminals.
+
```

Comment:
I think we can remove this; this is more of a bugfix that we need to respect `--no-color` on python 3.14.